linera-execution/src/wasm/mod.rs

```diff
@@ -214,6 +213,66 @@ impl UserServiceModule for WasmServiceModule {
     }
 }
 
+#[cfg(web)]
+const _: () = {
+    use js_sys::wasm_bindgen::JsValue;
+
+    impl TryFrom<JsValue> for WasmServiceModule {
+        type Error = JsValue;
+
+        fn try_from(value: JsValue) -> Result<Self, JsValue> {
+            // XXX: currently `Wasmer` is the only backend enabled on the Web.
```

Comment:
Was this meant to be TODO?  (I'd say we can remove because it's a bit obvious and not really low-level tech-debt)

Reply:
I have a task for this — I'll link it :)

The comment is here not because it's not obvious but because it's very easy to forget if we do add new backends on the Web, and would cause confusing runtime errors.